share/completions/brew.fish

```diff
@@ -6,10 +11,32 @@ function __fish_brew_needs_command
   return 1
 end
 
+function __fish_brew_needs_services_action
+  __fish_brew_using_command services
+
+  set cmd (commandline -opc)
+  if test (count $cmd) -le 2
+    return 0
+  end
+  return 1
+end
+
+function __fish_brew_services
+  brew services list | awk '{if (NR>1) print $1}'
```

Comment:
it just iterate all installed formulae and check if a `.plist` files exist in their root folder. I guess it's doable using fish but I wasn't sure if it's ok or not